tasks/adventure_solutions/task_5_loop_range_function.py

```diff
@@ -0,0 +1,69 @@
+import random
+import time
+
+def wrong_input(valid):
+    print(f"Sorry but the only valid options are: {valid}.")
+
+# create `dead` function with `death_message` argument
+def dead(death_message):
+    print(f"Sorry {name}, {death_message}.")
+    print("You're falling for...")
+
+    # print numbers slowly
+    for number in range(1,11):
+        print(number)
```

Comment:
I would have actually done with `range(10)` and `print(number +1)` but it is nice to see it in another way :)

Reply:
I don't have strong feelings about the way I did. if you prefer I could change it.

Reply:
it is totally fine